src/ResourceManagement/RecoveryServices/RecoveryServicesManagement/RecoveryServicesManagement.csproj

```diff
@@ -20,15 +20,54 @@
     <NoWarn>$(NoWarn);0108</NoWarn>
   </PropertyGroup>
   <ItemGroup>
-    <Compile Include="Generated\Models\**\*.cs" />
```

Comment:
Earlier was better way.
Adding each file name in csproj result in missing few files. Please revert back.